### PR TITLE
JOSS Paper

### PR DESCRIPTION
**Pull request by profxj**
Merges in JOSS paper in its current form and the substantial advances in docs (installation, step-by-step, and more).

This may not be the last word for JOSS, but I wish to get these doc improvements
into v1.1 and release that next week.

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -3,30 +3,93 @@
...
+
+  * Gemini/GNIRS
+  * Gemini/GMOS
+  * Gemini/FLAMINGOS 2
+  * Lick/Kast
+  * Magellan/MagE
+  * Magellan/Fire
+  * MDM/OSMOS
+  * Keck/DEIMOS (600ZD, 830G, 1200G)
+  * Keck/LRIS
```
Comment (rcooke-ast) [inline doc/index.rst:69]: Do you wish to add Keck KCWI?

File: doc/spectrographs.rst [hunk 2]
```diff
@@ -24,7 +24,7 @@ keck_lris_red_orig      Keck        :doc:`lris` spectrometer; red camera + origi
 keck_mosfire            Keck        MOSFIRE spectrometer; J and Y gratings tested
 keck_nires              Keck        NIRES spectrometer
 keck_nirspec_low        Keck        NIRSPEC spectrometer; low-dispersion
-keck_deimos             Keck        DEIMOS spectrometer (600ZD, 830G, 1200G)
+keck_deimos             Keck        :doc:`deimos` spectrometer (600ZD, 830G, 1200G)
```
Comment (rcooke-ast) [inline doc/spectrographs.rst:27]: And again, add KCWI?

**Review by rcooke-ast — APPROVED** [review]
Looks good - thanks for your efforts on this matter. My only suggestion is to include Keck KCWI... even though it's not "fully" supported, I'll be imminently reducing additional data soon that will basically cover all of the BM grating... up to you...

**Comment by profxj** [comment]
I suggest we list `Keck/KCWI` after its
next PR.  Thanks!

**Pull request merged**